ci/format.sh

```diff
@@ -32,6 +32,20 @@ SRC_DIR="$(cd "$SCRIPT_DIR/../.."; pwd -P)"
 DART_SDK_DIR="${SRC_DIR}/third_party/dart/tools/sdks/dart-sdk"
 DART="${DART_SDK_DIR}/bin/dart"
 
+# Check if a file named **/GeneratedPluginRegistrant.java exists in the project.
+# If it does, fail the build and print a message to the user pointing them to
+# the file and instructing them to remove it.
+#
+# See: https://github.com/flutter/flutter/issues/143782.
+GENERATED_PLUGIN_REGISTRANT_PATH=$(find "$SRC_DIR/flutter" -name "GeneratedPluginRegistrant.java")
+if [[ -n "$GENERATED_PLUGIN_REGISTRANT_PATH" ]]; then
+  echo "GeneratedPluginRegistrant.java found in the project."
+  echo "This file is generated by Flutter tooling and should not be checked in."
```

Comment:
It's not just that it shouldn't be checked in, it should be present at all right?

Reply:
Yeah better, changed!